### ISSUE TITLE
[B26] Advanced Break Seal can't handle multiple whitelist entries

### ISSUE THREAD

**Issue by praecipitator** [Azanor/thaumcraft-beta#1607]
If I specify two entries in an adv break seal's whitelist, it becomes non-functional. The golem doesn't break any block. If the list contains one block exactly, it works fine.

**Comment by VadersFist0501** [comment]
I didn't know that there was an advanced break seal... are you sure that it is not an addon?

**Comment by praecipitator** [comment]
Yes, I am. I'm pretty sure I didn't install any addons for TC. I don't even know of any for the current version.

**Comment by Lavaheart626** [comment]
Hey I'd like to bump this since it seems to still be an issue. I am using the MC eternal modpack which uses B26 too. At first I thought it was the ftb utils chunk claiming causing the golems to become non-operational. 
But it did turn out to be this issue here.

It's either that or the golems are so slow at checking their area that it seems like they're doing nothing

**Comment by praecipitator** [comment]
Dude, Thaumcraft is abandoned. This won't get fixed. Ever.

**Comment by Lavaheart626** [comment]
frick I didn't know that when I posted. That's a shame I finally figured out how to do all the essentia sorting too.
